website/src/pages/accelerate-dev-onboarding.tsx

```diff
@@ -0,0 +1,102 @@
+                        </h1>
+                        <h4 className="font-weight-bold">
+                            Self-serve onboarding, codebase exploration, and knowledge sharing
+                        </h4>
+                        <p>
+                            With Sourcegraph, developers can find their own answers without waiting for someone to point
+                            them to the relevant code. Get sharable links to help new developers ask specific questions
+                            with context included, and enable senior devs to share their knowledge without the friction
+                            of running local searches for specific bits of code.
+                        </p>
+
+                        <div className="mt-5">
+                            <div id="form-0" />
+                        </div>
+                    </div>
+
+                    <div className="col-lg-6 mt-4 mt-lg-0">
+                        <div className="embed-responsive embed-responsive-16by9">
+                            <iframe
+                                src="https://www.youtube.com/embed/DgwvhRW1Cbc"
+                                title="Accelerate developer onboarding with Sourcegraph"
+                                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                                allowFullScreen
+                                className="embed-responsive-item"
+                            />
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div className="bg-gradient-onahau-fog text-center">
+                <div className="container col-lg-6 py-6">
+                    <h2 className="mb-4 font-weight-bold">
+                        See how Convoy increases the efficiency and confidence of entry level developers
+                    </h2>
+                    <blockquote>
+                        <p>
+                            &ldquo;For our new developers, Sourcegraph has been invaluable to get to know the repository
+                            structure, to track down where code lives, and self-service during their
+                            investigations.&rdquo;
+                        </p>
+                        <figcaption className="text-muted mt-4">
+                            —Owen Kim, Senior Software Engineer at Convoy
+                        </figcaption>
+                    </blockquote>
+
+                    <img src="/external-logos/convoy-logo.svg" alt="Convoy logo" width={130} className="mt-4" />
+
+                    <Link to="/case-studies/convoy-improved-on-boarding" className="d-block mt-5 font-weight-bold">
+                        Read the full case study
+                    </Link>
+                </div>
+            </div>
+
+            <div className="text-center">
+                <div className="container col-lg-8 py-8">
```

Comment:
This looks like a smaller column on the design? 
![Screen Shot 2022-03-15 at 10 40 43 AM](https://user-images.githubusercontent.com/43454804/158402745-154ded84-a686-47fd-8f60-10916cafc7e4.png)